plugin.json

```diff
@@ -21,50 +21,50 @@
       {
         "key": "DisplayMode",
         "type": "radio",
-        "display_name": "Display the GIF as an",
+        "display_name": "Display the GIF as:",
         "default": "embedded",
         "options": [
           {
             "display_name": "Embedded image (the GIF cannot be collapsed)",
             "value": "embedded"
           },
           {
-            "display_name": "Collapsable image preview (the full URL is displayed, requires links preview to be activated)",
+            "display_name": "Collapsible image preview (the full URL is displayed, requires link previews to be enabled)",
             "value": "full_url"
           }
         ],
-        "help_text": "It is not yet possible to collapse an embedded image in Mattermost: use the Full URL option if preferred and keep an eye on [this issue](https://github.com/moussetc/mattermost-plugin-giphy/issues/12).\n\n**To activate links preview**, go to System Console > Posts > Enable Link Previews."
+        "help_text": "It is not yet possible to collapse an embedded image in Mattermost: use the Full URL option if preferred and keep an eye on [this issue](https://github.com/moussetc/mattermost-plugin-giphy/issues/12).\n\n To enable link previews, go to **System Console > Site Configuration > Posts > Enable Link Previews**."
       },
       {
         "key": "Provider",
         "type": "radio",
-        "display_name": "GIF Provider",
+        "display_name": "GIF Provider:",
```

Comment:
We might want to avoid trailing colons on the 3 radio options, which look a bit weird:
![Screenshot_20210117_222921](https://user-images.githubusercontent.com/8834237/104856551-a4426f00-5913-11eb-9d3b-59302333c823.png)

Reply:
Absolutely correct - I didn't register that they were radio options. My bad! :)